src/BatteryStateOfChargeService.cpp

```diff
@@ -1,5 +1,8 @@
 #include "BatteryStateOfChargeService.h"
-
+#include <iostream>
+#include <QTime>
+#include <QDebug>
+using namespace std;
```

Comment:
These 4 lines except for `#include <QTime>` are not being used and should be removed.